v/viridislite/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+   "maintainer": "stutiibm",
+   "package_name": "viridislite",
+   "github_url": "https://github.com/cran/viridislite",
+   "version": "0.4.2",
+   "default_branch": "master",
+   "build_script": "viridislite_ubi_9.3.sh",
+   "package_dir": "v/viridislite",
+   "docker_build": false,
+   "validate_build_script": true,
+   "use_non_root_user": false,
+   "*.*.*": {
```

Comment:
Generic regex. Can be made specifix to 0.*.* or 0.4.*

Reply:
Made required changes